contracts/RiskSteward/RiskStewardReceiver.sol

```diff
@@ -0,0 +1,322 @@
+     * @param updateType The type of update to set the config for
+     * @param debounce The debounce period for the update
+     * @param maxIncreaseBps The max increase bps for the update
+     * @param isRelative Whether the max increase bps is relative or absolute
+     */
+    function setRiskParameterConfig(
+        string calldata updateType,
+        uint256 debounce,
+        uint256 maxIncreaseBps,
+        bool isRelative
+    ) external onlyOwner {
+        riskParameterConfigs[updateType] = RiskParamConfig({
+            active: true,
+            debounce: debounce,
+            maxIncreaseBps: maxIncreaseBps,
+            isRelative: isRelative
+        });
+    }
+
+    /**
+     * @notice Gets the risk parameter config for a given update type
+     * @param updateType The type of update to get the config for
+     * @return The risk parameter config for the given update type
+     */
+    function getRiskParameterConfig(string calldata updateType) external view returns (RiskParamConfig memory) {
+        return riskParameterConfigs[updateType];
+    }
+
+    /**
+     * @notice Toggles the active status of a risk parameter config
+     * @param updateType The type of update to toggle the config for
+     */
+    function toggleConfigActive(string calldata updateType) external onlyOwner {
+        riskParameterConfigs[updateType].active = !riskParameterConfigs[updateType].active;
+    }
+
+    /**
+     * @notice Processes an update by its ID
+     * @param updateId The ID of the update to process
+     * @custom:event Emits RiskParameterUpdated with the update ID
+     * @custom:error Throws ConfigNotActive if the config is not active
+     * @custom:error Throws UpdateIsExpired if the update is expired
+     * @custom:error Throws ConfigAlreadyProcessed if the update has already been processed
+     * @custom:error Throws UpdateTooFrequent if the update is too frequent
+     * @custom:error Throws UpdateNotInRange if the update is not in range
+     * @custom:error Throws UnsupportedUpdateType if the update type is not supported
+     */
+    function processUpdateById(uint256 updateId) external whenNotPaused {
+        RiskParameterUpdate memory update = RISK_ORACLE.getUpdateById(updateId);
+        _validateUpdateStatus(update);
+        _processUpdate(update);
+    }
+
+    /**
+     * @notice Processes the latest update for a given parameter and market
+     * @param updateType The type of update to process
+     * @param market The market to process the update for
+     */
+    function processUpdateByParameterAndMarket(string memory updateType, address market) external whenNotPaused {
+        RiskParameterUpdate memory update = RISK_ORACLE.getLatestUpdateByParameterAndMarket(updateType, market);
+        _validateUpdateStatus(update);
+        _processUpdate(update);
+    }
+
+    function _processUpdate(RiskParameterUpdate memory update) internal {
+        if (Strings.equal(update.updateType, "MarketSupplyCaps")) {
```

Comment:
Would it make sense to use an enum for `updateType`?

Reply:
This type comes from Chaos Labs. Also so that we can make this extendable I think a string makes sense